### ISSUE TITLE
Country page: in a nutshell

### ISSUE THREAD

**Issue by hellais** [ooni/explorer#62]
In a nutshell should show greyed out text and icon for values set to none.

**Comment by sarathms** [comment]
Fixed in 4ce8d5b

**Comment by hellais** [comment]
I see that it always that "in a nutshell" section always says "OONI data confirms the blocking of websites" for the websites section, even though the returned JSON says `0`.

I believe this has to do with the fact that:
```
blockedWebsitesCount={100 || overviewStats.websites_confirmed_blocked}
```

So if `websites_confirmed_blocked` is zero (as is now the case for all countries) it will always say 100.

**Comment by sarathms** [comment]
```
blockedWebsitesCount={100 || overviewStats.websites_confirmed_blocked}
```
This was written this way to test the component in the absence of real data. This was fixed in c8e744b and should work as planned now.
